.github/workflows/comment-changes.yml

```diff
@@ -0,0 +1,34 @@
+name: comment-changes
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  aeruginous:
+    name: aeruginous
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.5.2
+        with:
+          fetch-depth: 0
+
+      - run: |
+          rustup update
+          cargo install aeruginous
+          aeruginous comment-changes -d ::= -o changelog.d/ -k -n 1
```

Comment:
I think we might need a cache here. I added `Swatinem/rust-cache@v2.4.0` to the main build pipeline, but I don't want if we can use it for installing crates as well. If not we can check [sccache crate](https://crates.io/crates/sccache) to achieve that.

I think we can get the segments from commit descriptions without a delimiter following a simple logic like:
- We split the first space part from the text using split_once() method in rust
- Then we check if the first part contains one of the keywords, and if not we can even assign a Default fallback value like Fixed or Changed
- You make the delimiter optional and the logic above as the fall back case when there are no delimiter

Do you think it would be possible & useful, to implement this suggestion in Aeruginous?

Reply:
Regarding the cache, I would recommend the official caching Action, `actions/cache`.  However, there are some open issues on caches not being restored but some of the users who commented assumed that the failure would have originated from different branches on which the caches were created and attempted to be restored; this would not be the case, here, so I expect that this should not cause problems with this use case.

Reply:
I think we can leave it for now without caching and see how much time it takes to install aeruginous, then we can decide if we need to add some caching to it

Reply:
We could also use this Action:  https://github.com/baptiste0928/cargo-install.

Another repository using Aeruginous for fragment creation also applies the linked Action to speed up the installation.  The Action will cache the dependencies, so it will be definitely worth a try.  The time is reduced there to about one minute for Aeruginous v2.1.0.

Reply:
Thanks for bringing this up :+1: 
I think we can definitely use this since rust built time isn't the best